errext/format.go

```diff
@@ -0,0 +1,28 @@
+package errext
+
+import (
+	"errors"
+)
+
+// Format formats the given error as a message (string) and a map of fields.
+// In case of Exception, it uses the stack trace as the error message.
+// In case of HasHint, it also adds the hint as a field.
```

Comment:
```suggestion
// In case of [Exception], it uses the stack trace as the error message.
// In case of [HasHint], it also adds the hint as a field.
```

This should make this link within godoc